src/components/VideoDetail/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Styled from './styled';
+import VideoCard from '../VideoCard';
+
+const VideoDetail = ({ items, current, setCurrent }) => {
+  const showVideoDetail = (item) => {
+    console.log(item);
+    setCurrent(item);
+  };
+
+  return (
+    <Styled.Container>
+      <Styled.VideoContainer>
+        <Styled.VideoPlayerContainer>
+          <Styled.VideoPlayer
+            height="315"
+            width="560"
+            src={`https://www.youtube.com/embed/${current.id.videoId}`}
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+            title="current.snippet.title"
+          />
+        </Styled.VideoPlayerContainer>
+        <Styled.Title>{current.snippet.title}</Styled.Title>
+        <Styled.Description>
+          <p>{current.snippet.description}</p>
+        </Styled.Description>
+      </Styled.VideoContainer>
+      <Styled.Side>
+        {items
+          .filter(
+            (item) =>
+              item.id.kind === 'youtube#video' && item.id.videoId !== current.id.videoId
+          )
+          .map((item) => (
+            <Styled.VideoScreenshotContainer>
+              <VideoCard
+                onClick={() => showVideoDetail(item)}
+                key={item.id.videoId}
+                title={item.snippet.title}
+                url={item.snippet.thumbnails.medium.url}
+              />
+            </Styled.VideoScreenshotContainer>
```

Comment:
Remember that the key should go on the root element:

```suggestion
            <Styled.VideoScreenshotContainer key={item.id.videoId}>
              <VideoCard
                onClick={() => showVideoDetail(item)}
                title={item.snippet.title}
                url={item.snippet.thumbnails.medium.url}
              />
            </Styled.VideoScreenshotContainer>
```

Reply:
Hey tocayo @jparciga, yes I know, I am behind on the tests, need to work on that.

Reply:
No rush, just let me know once you have them 👍🏻